app-editors/vim/vim-8.2.0111.recipe

```diff
@@ -10,11 +10,11 @@ configuration files.
 Despite this, Vim can be configured to work in a very simple (Notepad-like) \
 way, called evim or Easy Vim."
 HOMEPAGE="https://www.vim.org/"
-COPYRIGHT="1991-2019 Bram Moleenar et al."
+COPYRIGHT="1991-2020 Bram Moleenar et al."
 LICENSE="Vim"
-REVISION="3"
+REVISION="4"
```

Comment:
```suggestion
REVISION="0"
```

The `REVISION` variable is used to differentiate between different versions of the same recipe. Since you're updating the package version, `REVISION` should be reset to `0`.

Reply:
Thank you for your input, made changes as you've indicated.

About the review, no worries, just wanted to make sure it is not neglected :). Thank you for the clarification.